server/middlewares/controllers/authorController.js

```diff
@@ -6,4 +6,25 @@ const addAuthor = async (req, res) => {
   return res.status(201).jsend(author);
 };
 
-module.exports = { addAuthor };
+const getAuthor = async (req, res) => {
```

Comment:
fixed

Reply:
Let's use the `id` to get a single author. This is because there could be authors with the same name. Also, this should be `author`. Remember it's called `getSingleAuthor`. You can use this to get the author:

```
await Author.query().findById(id)
```